javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -1131,6 +1131,14 @@ class Options {
       expiry = Math.floor(date.getTime() / 1000);
     }
 
+    // if not declared, attribute is set to 'undefined'
+    if(typeof sameSite === "undefined") {
+      // do nothing
```

Comment:
if you do nothing you could also just remove that part (meaning the whole if) 😉

Reply:
if i dont add any cookie like below:

    await driver.manage().addCookie({name:'key', value: 'value'});

SameSite value is set to 'undefined'. Removing the if condition, it expects sameSite value should be 'Lax' or 'Strict'  as it is now undefined-> InvalidArgumentError: Invalid sameSite cookie value 'undefined'. It should be either "Lax" (or) "Strict" 

But it is the users choice of adding sameSite cookie property. if it is by default should be either 'Lax' or 'Strict'. 

Do we have any value @christian-bromann, because chrome sets value to undefined as i have observed.

Suggest me!

Reply:
just remove it:
```js
    if(typeof sameSite === "undefined") {
      // do nothing
    if(sameSite !== 'Strict' && sameSite !== 'Lax') {
      throw new error.InvalidArgumentError(
          `Invalid sameSite cookie value '${sameSite}'. It should be either "Lax" (or) "Strict" `);
    }
```
is the same as
```js
    if(sameSite !== 'Strict' && sameSite !== 'Lax') {
      throw new error.InvalidArgumentError(
          `Invalid sameSite cookie value '${sameSite}'. It should be either "Lax" (or) "Strict" `);
    }
```

Reply:
Ah I think I understand, then just do:
```js
    if(sameSite && sameSite !== 'Strict' && sameSite !== 'Lax') {
      throw new error.InvalidArgumentError(
          `Invalid sameSite cookie value '${sameSite}'. It should be either "Lax" (or) "Strict" `);
    }
```

Reply:
Done! :)